src/SideciTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Yoshihiro
+ * Date: 4/22/16
+ * Time: 11:24 PM
+ */
+
+namespace OhSky\LineTrialBot;
+
+
+class SideciTest{
+
+    public $name='';
+
+    public function testFunction() {
+        if(false)return true;
```

Comment:
__[code_sniffer]__
* Expected 1 space after IF keyword; 0 found
* Inline control structures are not allowed